model/incoming_webhook.go

```diff
@@ -112,6 +114,14 @@ func (o *IncomingWebhook) IsValid() *AppError {
 		return NewAppError("IncomingWebhook.IsValid", "model.incoming_hook.description.app_error", nil, "", http.StatusBadRequest)
 	}
 
+	if len(o.PostUsername) > 128 {
```

Comment:
Why this max lenght? Same question for the icon url?

Reply:
When I had looked into the corresponding maximum length in the user store, I thought I saw `128`, but upon closer inspection, it looks like the maximum size is only `64`. I'll fix this to be consistent -- thanks for spotting this!

Reply:
I picked the length of `1024` for the `PostIconURL` to match other `*URL` lengths in various stores, e.g.: https://github.com/mattermost/mattermost-server/blob/master/store/sqlstore/command_store.go#L30

I've typically picked larger constraints for URL fields, but opted for consistency here. Let me know if you think that's the right approach :)